images/planet-dump/start.sh

```diff
@@ -30,7 +30,8 @@ osmosis --read-apidb \
 # AWS
 if [ $CLOUDPROVIDER == "aws" ]; then
 	# Save the path file
-	echo "https://$AWS_S3_BUCKET.s3.amazonaws.com/planet/$planetPBFFile" > $stateFile
+	AWS_URL=${AWS_S3_BUCKET/s3:\/\//http:\/\/}
+	echo "$AWS_URL.s3.amazonaws.com/planet/$planetPBFFile" > $stateFile
```

Comment:
@Rub21 any reason why we switched from s3 url to http? Also why not https?

Reply:
That was ,because Dan , asked to change  that to http.